### DISCUSSION TITLE
Update FlashList layout on data change

### DISCUSSION THREAD

**Discussion by Pietro-Putelli** [Shopify/flash-list#716] [General]
Hi everyone, I'm using FlashList for different purposes in my app and I'm struggling with layout update.
If, for example I refresh a list and remove some items, it remains a blank space and the layout is not updated.

Do I need to use invoke these methods every time I change the data prop hence update the layout of the list?

```
listRef.current?.prepareForLayoutAnimationRender();
LayoutAnimation.configureNext(LayoutAnimation.Presets.easeInEaseOut);
```
Or is there an alternative solution?
Thank you

**Comment by naqvitalha** [comment]
The layout animation methods help with layout animations. There's no need to call them when updating data.
Can you provide a sample on expo for me to look at? You are not required to do anything special for updating data
